arangod/Graph/PathManagement/PathValidatorTracer.h

```diff
@@ -77,8 +86,19 @@ class PathValidatorTracer {
   void unpreparePruneContext();
   void unpreparePostFilterContext();
 
- private:
-  PathValidatorImplementation _impl;
+  auto setForbiddenVertices(std::unique_ptr<VertexSet> forbidden)
+      -> void requires HasForbidden<PathValidatorImplementation> {
+    auto copy = std::make_unique<VertexSet>(*forbidden);
+    _impl.setForbiddenVertices(std::move(copy));
```

Comment:
Why have a copy here? And no need to move, since unique_ptr does not have copy ctor
```suggestion
    _impl.setForbiddenVertices(forbidden);
```

Reply:
Fixed with the move to shared_ptr